packages/graph/hash_graph/lib/graph/src/api/rest/data_type.rs

```diff
@@ -67,16 +69,14 @@ struct CreateDataTypeRequest {
     ),
     request_body = CreateDataTypeRequest,
 )]
-async fn create_data_type<S: Store>(
+async fn create_data_type<S: Store + Send + Sync>(
```

Comment:
I would say this bound is very tedious to have at every route, but I assume this will disappear relatively soon, which is why I'm letting it be